doorway-ui-components/src/global/mixins.scss

```diff
@@ -71,15 +71,15 @@
   &.is-primary {
     background-color: var(--primary-appearance-background-color, var(--bloom-color-primary));
     border-color: var(--primary-appearance-border-color, var(--bloom-color-primary));
-    color: var(--primary-appearance-label-color, var(--bloom-color-white));
+    color: var(--primary-appearance-label-color, var(--bloom-color-blue-300));
```

Comment:
For reference this is the color of the text inside the buttons. In the mocks it's blue-300, not white